File: Bubble sort/Bubble sort.java
public class BubbleSort {
	public static void main(String[] args) {
		int[] arr= {4,5,2,7,8,1,9,10,3};
		int len=arr.length;
		
		for(int i=len-1; i>0; i--) {
			for(int j=0; j<i; j++) {
				if(arr[j]>arr[j+1]) {
					int temp=arr[j];
					arr[j]=arr[j+1];
					arr[j+1]=temp;
				}
			}
		}
	}
}

File: Bubble sort/Bubble sort.cpp
#include <iostream>

int main(void)
{
	int arr[10] = { 33, 2, 45, 17, 100, 22, 22, 3, 0, 12 };

	for (int i = 0; i < 10; i++){
		for (int j = 0; j < 10 - 1-i; j++){
			if (arr[j] > arr[j + 1]) std::swap(arr[j], arr[j + 1]);
		}
	}

	for (int i = 0; i < 10; i++) std::cout << arr[i] << " ";

	return 0;
}